documentation/API-design-guidelines.md

```diff
@@ -1326,25 +1327,52 @@ If this capability is present in CAMARA API, `webhook` object attribute **must**
 
 | attribute name | type | attribute description | cardinality |
 | ----- |	-----  |	 -----  | -----  | 
-| notificationUrl | string | https callback address where the notification must be POST-ed | mandatory |
-| notificationAuthToken | string | OAuth2 token to be used by the callback API endpoint. It MUST be indicated within HTTP Authorization header e.g. Authorization: Bearer $notificationAuthToken | optional |
+| sink | string | https callback address where the notification must be POST-ed | mandatory |
+| sinkCredential | object | Sink credential provides authentication or authorization information necessary to enable delivery of events to a target. In order to be updated in future this object is polymorphic | optional |
 
-##### Instance-based (implicit) subscription example
+Several types of `sinkCredential` could be available in future but for now only access token credential are managed.
 
-```javascript
+``sinkCredential`` attributes table (must be access token for now):
+
+| attribute name | type | attribute description | cardinality |
+| ----- |	-----  |	 -----  | -----  | 
+| credentialtype | string | Type of the credential - MUST be set to `ACCESSTOKEN` for now | mandatory |
+| accessToken | string | Access Token granting access to the POST operation to create notification | mandatory |
+| accessTokenExpireUtc | string date-time | An absolute UTC instant at which the access token shall be considered expired. | mandatory |
+| accessTokenType | string | Type of access token - MUST be set to `Bearer` for now | mandatory |
+
+
+##### Instance-based (implicit) subscription examples
+
+If credentials are not required:
```

Comment:
Referring to issue https://github.com/camaraproject/Commonalities/issues/204
Should we consider empty security scheme (even it is the risk of API Consumer)?

Reply:
No.

Reply:
This is something that we need to discuss. 
I prefer having such discussion within issue 204. It has collateral considerations.
Would also like to know @shilpa-padgaonkar and @bigludo7 view on that

Reply:
I would prefer to discuss it in 204 & not block this PR for it.